client/store.js

```diff
@@ -250,7 +250,22 @@ store.setupKresus = function(cb) {
         data.StaticBanks = world.banks;
 
         has(world, 'categories');
+        // Ensure every category has a color
+        let titleToColor = function(str) {
```

Comment:
I'd prefer that this function be a helper in `models.js`, and used this way in Category's ctor:

`this.color = (maybeHas(arg, 'color') && arg.color) || titleToColor(this.title);`